apps/meteor/tests/e2e/register.spec.ts

```diff
@@ -136,6 +137,39 @@ test.describe.parallel('register', () => {
 		});
 	});
 
+	test.describe('Registration form validation - username', async () => {
+		test.beforeEach(async ({ page }) => {
+			poRegistration = new Registration(page);
+			poUtils = new Utils(page);
+		});
+		test('should not allow registration with existing username', async ({ page }) => {
+			const username = faker.internet.userName();
+			await request.post('/api/v1/users.register').set('Content-Type', 'application/json').send({
```

Comment:
Instead of registering a new user, you can use one of the existing test users fixtures 
https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/tests/e2e/fixtures/inject-initial-data.ts#L11

Theses users are registered whenever the server is set up in test mode